src/mappings/geyser.ts

```diff
@@ -231,6 +236,19 @@ export function handleRewardsDistributed(event: RewardsDistributed): void {
 }
 
 
+export function handleOwnershipTransferred(event: OwnershipTransferred): void {
+  let geyser = Geyser.load(event.address.toHexString());
+  let newOwner = User.load(event.params.newOwner.toHexString());
+  if (newOwner == null) {
+    newOwner = createNewUser(event.params.newOwner);
```

Comment:
this new owner needs to be saved out, currently causing a query error if the new owner didn't exist before
```
      "message": "Null value resolved for non-null field `owner`"
```